Css Refactor

ghislaineguerin opened the pull request:
This pull request fixes the following issues:
- Specific and/or nested CSS selectors that prevented new content from being added were removed
- Components that happened in multiple pages are now reusable includes

What's missing:
- Must create utility classes to apply background effects to any component
- Blog refactor
- Hero section refactor
- Improved circle background CSS

The main goal of these fixes is preventing the spread of technical debt onto new pages. 

@wadie I think the JS errors on the build will be fixed once we merge your changes.

marcoow commented on dd-wrt/index.html:
```diff
@@ -154,56 +98,29 @@ <h1 class="jumbotron-header">
+
+<script type="text/javascript">
+  $(function() {
+    $('.card-expertise').matchHeight();
```
Can we solve this in CSS instead? I't like to not depend on jQuery if possible.

ghislaineguerin replied:
@marcoow flexbox would work if the components are all on the same container, but I feel it will restrict some layout possibilities. Would a pure JS solution work? I've chosen this one because bootstrap requires jquery anyways.

marcoow replied:
We shouldn't be using anything from Bootstrap that requires jQuery really. I'd want this to be as light weight as possible. Can we move everything into the same container for now so we can use flexbox? This is only an intermediate solution anyways after all…

ghislaineguerin replied:
@marcoow that makes sense, I think making it lightweight is the priority at this point, I can solve this with CSS without the need to modify containers.

ghislaineguerin commented:
@marcoow I have fixed all the points you've listed.
- For the logos, I've removed the need to have both color and b&w version loading, which improved loading time. 
- I've solved the cards matching height w/ CSS, looks like I could solve it with flexbox, so we are good.
- As for the clickable cards, I originally eliminated that because the way it's done is not accessible, even for the case study cards I'm not 100% happy with the markup. See this as reference: https://inclusive-components.design/cards/
So if ok with you, I'd like to work on an accessible way to link cards and I think we are going to be good to release after that.


Overall improvement looks really nice:
<img width="598" alt="screen shot 2018-11-22 at 12 06 38 pm" src="https://user-images.githubusercontent.com/845767/48899166-195aac80-ee4f-11e8-81fb-c80509901840.png">

![image](https://user-images.githubusercontent.com/845767/48898890-507c8e00-ee4e-11e8-9ce2-9dab464aa424.png)

ghislaineguerin commented:
@marcoow I've run the accessibility audit and it seems we are doing fine now. I'm going to hide the long paragraphs for screen readers and added descriptive aria labels to links. I have fixed the other issues and I'm hoping we are close to releasing. Let me know once you review again.
